hi-ml-histopathology/src/SSL/datamodules_and_datasets/cifar_datasets.py

```diff
@@ -0,0 +1,30 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+
+from torchvision.datasets import CIFAR10, CIFAR100
+
+from SSL.datamodules_and_datasets.dataset_cls_utils import DataClassBaseWithReturnIndex
+
+
+class HIMLCIFAR10(DataClassBaseWithReturnIndex, CIFAR10):
```

Comment:
I'm a fan of the .NET naming convention that says "all three-or-more letter acronyms should be Pascal cased", so here it would be HimlCifar10. Particularly help if two acronyms clash like here